site/standalone-setup.md

```diff
@@ -62,7 +62,19 @@ fluxctl identity
 Create a Kubernetes Secret from a private key:
 
 ```sh
-kubectl create secret generic flux-git-deploy --from-file=identity=/path/to/private_key
+kubectl create secret generic flux-git-deploy --from-file=identity=/full/path/to/private_key
+```
+
+this will result in a secret that has the structure:
```

Comment:
I'm not against this change, but I am curious what prompted it.

Reply:
it was prompted by the other instruction `kubectl create secret generic flux-git-deploy --from-file /path/to/identity` in site/fluxctl.md not naming the argument (which then depends on the path being called `identity` so that the data key will be named `identity`.  I didn't catch this when I was following the instructions  and it caused  a lot of unnecessary frustration.  Eventually, while debugging I started inspecting the raw secret and the obvious question came forward: what data key is flux even looking for?  fast forwarding: I eventually figured it out because I saw other commands (like this one, actually) that were naming the argument to prevent confusion. I made sure to then go and make any other references to this action in the docs consistent with each other and that's where this one comes from.

Reply:
I see, in `--from-file=/path/identity` it's not clear that the base name of the file _must_ be `identity` for it to work. Giving the example of the result (to some extent) reinforces that constraint.

Reply:
exactly